packages/builder/src/builder.js

```diff
@@ -44,7 +44,7 @@ export default class Builder {
       restart: null
     }
 
-    this.supportedExtensions = ['vue', 'js', ...(this.options.build.additionalExtensions || [])]
+    this.supportedExtensions = ['vue', 'js', ...(this.options.extensions || []), ...(this.options.build.additionalExtensions || [])]
```

Comment:
Are these options identical now? /cc @kevinmarrec

Reply:
Please let me know if I should use nuxt.options.build.additionalExtensions in the module that add support for coffeescript extension.
Like this.nuxt.options.build.additionalExtensions.push ('coffee')

Reply:
@pi0 No
https://github.com/nuxt/typescript/blob/master/packages/typescript-build/lib/module.js#L18
We need to be aware that the 2 options aren't used at the same places in core

Reply:
`options.extensions` is more or less related to runtime, meanwhile the others only buildtime